kobo/urls.py

```diff
@@ -8,15 +8,29 @@
 from django.views.generic.base import RedirectView
 from rest_framework import status
 from rest_framework.exceptions import server_error
+from drf_spectacular.views import (
+    SpectacularAPIView,
+    SpectacularRedocView,
+    SpectacularSwaggerView,
+)
 
 from kpi.utils.urls import is_request_for_html
 
 admin.autodiscover()
-admin.site.login = staff_member_required(
-    admin.site.login, login_url=settings.LOGIN_URL
-)
+admin.site.login = staff_member_required(admin.site.login, login_url=settings.LOGIN_URL)
 
 urlpatterns = [
+    path('api/schema/', SpectacularAPIView.as_view(), name='schema'),
+    path(
+        'api/schema/swagger/',
+        SpectacularSwaggerView.as_view(url_name='schema'),
+        name='swagger-ui',
+    ),
+    path(
+        'api/schema/redoc/',
+        SpectacularRedocView.as_view(url_name='schema'),
+        name='redoc',
+    ),
```

Comment:
I just copied the example from docs for a quick PoC. Questions:
1. what api path you'd like?
2. is this the right place to add it to urls?
3. let's use swagger or redoc, or both? My preference is swagger.